### PR TITLE
better online manual

### PR DESCRIPTION
**Pull request by eylles**
i pulled the trigger into opening this, may take the time later to pretty up the online manual, perhaps a dark theme and make the links a navigation sidebar.

### CODE REVIEW

**Comment by N-R-K** [comment]
You can open PRs from other people's repo/branch? Hmm, didn't know that.

File: update.sh
Comment (N-R-K) [inline update.sh:3]: We don't really need to store `index.html` , it can be piped into the awk command directly.

File: update.sh
Comment (N-R-K) [inline update.sh:4]: I don't think we need to delete the existing style, just appending our style at the end should overwrite the default.

**Review by N-R-K — CHANGES_REQUESTED** [review]

**Comment by N-R-K** [comment]
@GRFreire not able to push my changes for some reason. Anyways can you update the script to this?

```sh
git show "master:nsxiv.1" | groff -mandoc -Thtml |
  sed 's|^</head>|<link rel="stylesheet" href="index.css"></head>|' > index.html
```

**Comment by N-R-K** [comment]
> perhaps a dark theme and make the links a navigation sidebar.

I'm okay with a dark theme. A sidebar might be more trouble than it's worth, remember that the html will be generate by groff.

**Comment by GRFreire** [comment]
Tried to create a PR and realized someone have already did it. Sorry for not seeing this earlier.

@N-R-K I didnt create the PR, so I cant open for changes (even though it is on my fork), but I'll update for you.

File: index.css
Comment (N-R-K) [inline index.css:6]: The font appears a bit small for me, can we use 1.0 or atleast 0.95 something?

```suggestion
  font-size: 1.0rem;
```

**Comment by GRFreire** [comment]
What do you think about adding a favicon?
This should do the job:

**Comment by N-R-K** [comment]
> What do you think about adding a favicon? This should do the job: 

No strong opinion. Do what you feel is best.

File: update.sh
Comment (N-R-K) [inline update.sh:4]: This should point to our repo.
```suggestion
  sed 's|^</head>|<link rel="stylesheet" href="index.css"><link rel="icon" type="image/png" href="https://raw.githubusercontent.com/nsxiv/nsxiv/master/icon/16x16.png" sizes="16x16"></head>|' > index.html
```

**Comment by GRFreire** [comment]
Added the favicon so we can check for any performance loss.

> No strong opinion. Do what you feel is best.

Me neither, just a thing that poped into my head. We can undo this commit anytime.

File: update.sh
Comment (N-R-K) [inline update.sh:4]:
```suggestion

css='<link rel="stylesheet" href="index.css">'
icon_src='https://raw.githubusercontent.com/nsxiv/nsxiv/master/icon/16x16.png'
icon='<link rel="icon" type="image/png" href="'"$icon_src"'" sizes="16x16">'

git show "master:nsxiv.1" | groff -mandoc -Thtml |
  sed 's|^</head>|'"${css}${icon}"'</head>|' > index.html
```

Comment (N-R-K) [reply update.sh:4]: Untested, but I think it should work. @GRFreire

Comment (GRFreire) [reply update.sh]: That's way cleaner, thanks

**Review by N-R-K — APPROVED** [review]

**Comment by N-R-K** [comment]
@GRFreire Feel free to merge this when you're done. Maybe wait for @eylles 's review though.

**Comment by GRFreire** [comment]
I'll wait for @eylles and see if anyone else has anything to say

File: index.css
Comment (N-R-K) [inline index.css:4]: We already have `color` at bottom.
```suggestion
```

File: update.sh
Comment (N-R-K) [inline update.sh:3]: I think `style.css` is more common. @GRFreire could you rename the `.css` ?
```suggestion
css='<link rel="stylesheet" href="style.css">'
```

**Review by N-R-K — CHANGES_REQUESTED** [review]

**Review by N-R-K — APPROVED** [review]

**Comment by eylles** [comment]
so since this has 14 commits we squash it right?

**Pull request merged**
